app/src/main/res/layout-sw600dp-land/add_profile_activity.xml

```diff
@@ -212,9 +214,8 @@
                 android:id="@+id/add_profile_activity_allow_download_switch"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:enabled="@{viewModel.validPin ? true : false}"
-                android:minWidth="48dp"
-                android:minHeight="48dp"
+                android:clickable="false"
```

Comment:
Also, why are we preventing a SwitchCompat from clicking?

Reply:
Because the entire item has been made clickable so the result will be same for user. But if we make this clickable along with the entire container then we will need to copy paste the code which should be executed after the user action which is not the correct way.

From user perspective switch is always clickable.